applications/dashboard/modules/class.headmodule.php

```diff
@@ -525,8 +525,8 @@ public function toString() {
                 $tag = $attributes[self::TAG_KEY];
 
                 // Inline the content of the tag, if necessary.
-                if (val('_hint', $attributes) == 'inline') {
-                    $path = val('_path', $attributes);
+                if (($attributes['_hint'] ?? false) == 'inline') {
```

Comment:
I have high confidence in this change not breaking anything because $attributes is already later accessed as an array.